backend/src/acidwatch_api/models/gibbs_minimization_model.py

```diff
@@ -19,6 +19,7 @@ class _EquationOfState(StrEnum):
     SRK = "SRK"
     PR = "PR"
     SRKCPA = "SRKCPA"
+    IdealGas = "Ideal Gas"
```

Comment:
I think it will be nice to keep consistent capitalization. What about "IG" instead of "IdealGas"?